curriculum/challenges/english/25-front-end-development/quiz-dom-manipulation-and-click-event-with-javascript/66edd07682767adff3a6231e.md

```diff
@@ -17,439 +17,560 @@ Answer all of the questions below correctly to pass the quiz.
+target.remove(target.lastElementChild);
+target.appendChild("That's it");
+```
 
 ---
 
-Placeholder distractor 2
+```js
+const target = document.getElementById("target");
+target.lastElementChild.replaceWith("That's it");
+```
 
 ---
 
-Placeholder distractor 3
+```js
+const target = document.querySelector("p#target");
+target.replaceChild("That's it", target.lastChild);
+```
 
 #### --answer--
 
-Placeholder answer
+```js
+const target = document.getElementById("target");
+target.removeChild(target.lastChild);
+target.append("That's it.");
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which one of these affirmations is false?
 
 #### --distractors--
 
-Placeholder distractor 1
+The `innerHTML` element's property returns all the HTML content inside it as text.
 
 ---
 
-Placeholder distractor 2
+The `innerText` element's property only returns the text content that is shown on screen.
 
 ---
 
-Placeholder distractor 3
+The `textContent` node's property returns all the text content inside it.
 
 #### --answer--
 
-Placeholder answer
+The `htmlContent` node's property only returns HTML content that is shown on screen.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following JavaScript lines do not insert a thematic break at the end of the document body?
 
 #### --distractors--
 
-Placeholder distractor 1
+```js
+document.body.insertBefore(document.createElement("hr"), null);
+```
 
 ---
 
-Placeholder distractor 2
+```js
+document.body.appendChild(document.createElement("hr"));
+```
 
 ---
 
-Placeholder distractor 3
+```js
+document.body.after(document.createElement("hr"));
+```
 
 #### --answer--
 
-Placeholder answer
+```js
+document.body.append("<hr/>");
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following statements is false?
 
 #### --distractors--
 
-Placeholder distractor 1
+`window` is an alias for `navigator.window
+
+```js
+window === navigator.window // returns true
+```
 
 ---
 
-Placeholder distractor 2
+You can use the user screen properties (like position and size) using the `Window` object.
 
 ---
 
-Placeholder distractor 3
+`navigation` is an alias for `window.navigation`
+
+```js
+navigation === window.navigation // returns true
+```
 
 #### --answer--
 
-Placeholder answer
+You can access the clipboard using :
+
+```js
+await browser.clipboard.read();
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Assuming the JavaScript variable `element` is an HTML `Element` object, which code does not do the same thing as the others?
 
 #### --distractors--
 
-Placeholder distractor 1
+```js
+element.toggleAttribute("disabled");
+```
 
 ---
 
-Placeholder distractor 2
+```js
+element.hasAttribute("disabled") ? element.removeAttribute("disabled") : element.setAttribute("disabled");
+```
 
 ---
 
-Placeholder distractor 3
+```js
+element.toggleAttribute("disabled", element.!hasAttribute("disabled"));
```

Comment:
```suggestion
element.toggleAttribute("disabled", !element.hasAttribute("disabled"));
```
Otherwise there's `SyntaxError`.

Reply:
Yes, that's an intended `SyntaxError` for it to be a distractor, otherwise it would be an other working solution.

Reply:
But the code should be working and toggling the attribute 😛. Question is asking for option that's not doing it.

Reply:
You are right, my bad :') Nice double check !

Reply:
I've been a little bit too confident on this one sorry ! 😅

Reply:
I also noticed I didn't provide a second argument to SetAttribute, causing a TypeError, I got this fixed